### ISSUE TITLE
Can not pass `--config` parameter to documentation.

### ISSUE THREAD

**Issue by jamestalmage** [documentationjs/documentation-readme#12]
I have tried every variation I can think of:
```
$ documentation-readme -s "API" -- foo.js -c ./docs.json
$ documentation-readme -s "API" -- foo.js --config ./docs.json
$ documentation-readme -s "API" -- foo.js --config=./docs.json
```
Also with `foo.js` last, and wrapping stuff in `"`.

I get the following error.

```
Cannot find module /.../--config=./docs.json from ...
```

and similar no matter what I try.

**Comment by anandthakker** [comment]
@jamestalmage Thanks for the report.  Should be fixed in 2.1.1 -- can you give it a try and let me know?

**Comment by jamestalmage** [comment]
:+1:
